### PR TITLE
Fix image generation bug

### PR DESCRIPTION
**Pull request by Huuuaiyuan**
1, add all the new configuration files
2, fix image generation bugs

### CODE REVIEW

File: src/lib/CalibrationVerification.cpp [hunk 1]
```diff
@@ -601,8 +601,8 @@ void CalibrationVerification::SaveCameraVisuals() {
       LoadLookupTree();
 
       // load image from bag
-      std::shared_ptr<cv::Mat> current_image = GetImageFromBag(
-          params_->camera_params[measurement->camera_id]->topic);
+      std::shared_ptr<cv::Mat> current_image = std::make_shared<cv::Mat>();
+      *current_image = measurement->img;
```
Comment (nickcharron) [inline src/lib/CalibrationVerification.cpp:605]: I don't think this even needs to be a pointer, can you change it to just a cv::Mat?

Comment (nickcharron) [reply src/lib/CalibrationVerification.cpp:605]: Also, can we remove the GetImageFromBag function now?

**Review by nickcharron — CHANGES_REQUESTED** [review]
I think I mentioned before I don't want to merge these config files. I only want this repo to contain example configs, otherwise it'll get too cluterred and confusing for people trying to use the code. Just save your config files locally

After you remove the config file changes, I will approve the PR. Please see my other comments though

**Comment by Huuuaiyuan** [comment]
Sure, I will.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: src/lib/CalibrationVerification.cpp (hunk 1 above)
Comment (Huuuaiyuan) [reply src/lib/CalibrationVerification.cpp:605]: > Also, can we remove the GetImageFromBag function now?

Removed

**Pull request merged**